synfig-studio/src/synfigapp/wplistconverter.cpp

```diff
@@ -101,8 +101,12 @@ WPListConverter::operator()(std::list<synfig::WidthPoint> &wp_out, const std::li
 		p1=p2;
 	}
 	// Calculate the normalized cumulative distances
-	for(i=0;i<n;i++)
-		norm_distances.push_back(distances[i]/distances[n-1]);
+	if (n == 1 && synfig::approximate_zero(distances[n-1])) {
+		norm_distances.push_back(0.);
+	} else {
+		for(i=0;i<n;i++)
+			norm_distances.push_back(distances[i]/distances[n-1]);
```

Comment:
What about other points? 
Can this issue affect other points? Like close distance between points 3-4 or something?

Reply:
The problem is the last cumulative distance (distances[n-1]) being zero, as it is denominator.

Reply:
Do you think I should remove the "n == 1" check?

Reply:
DOne